cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationOperations.java

```diff
@@ -19,20 +33,180 @@ public ApplicationOperations(DefaultCloudFoundryOperations cloudFoundryOperation
     }
 
     public List<ApplicationBean> getAll() {
-        List<ApplicationSummary> applications = this.cloudFoundryOperations.applications().list().collectList().block();
+        List<ApplicationSummary> applications = this.cloudFoundryOperations
+                .applications()
+                .list()
+                .collectList()
+                .block();
 
         // create a list of special bean data objects, as the summaries cannot be serialized directly
         List<ApplicationBean> beans = new ArrayList<>();
         for (ApplicationSummary summary : applications) {
-            GetApplicationManifestRequest manifestRequest = GetApplicationManifestRequest.builder()
-                    .name(summary.getName()).build();
-            ApplicationManifest manifest = this.cloudFoundryOperations.applications()
-                    .getApplicationManifest(manifestRequest).block();
+            ApplicationManifest manifest = getApplicationManifest(summary);
 
             beans.add(new ApplicationBean(manifest));
         }
 
         return beans;
     }
 
+    private ApplicationManifest getApplicationManifest(ApplicationSummary applicationSummary) {
+        return this.cloudFoundryOperations
+                .applications()
+                .getApplicationManifest(GetApplicationManifestRequest
+                        .builder()
+                        .name(applicationSummary.getName())
+                        .build())
+                .block();
+    }
+
+    /**
+     * TODO: Clarification with project owner necessary:
+     * TODO: how to proceed when push fails to apply some settings?
+     * TODO: remove on fail ?
+     * TODO: keep on fail and only print errors as warnings ?
+     *
+     * for now keep on fail with error messages
+     */
+    /**
+     *
+     *  Pushes the app to the cloud foundry instance specified within the cloud foundry operations instance
+     *
+     * @param bean  application bean
+     * @param noStart   if the app should not start after being created
+     * @throws CreationException
+     */
+    public void create(ApplicationBean bean, boolean noStart) throws CreationException {
+        checkNotNull(bean.getName());
+        checkNotEmpty(bean.getName());
+        checkNotNull(bean);
+        //otherwise IllegalArgumentException is thrown within reactor stream which is difficult to handle
+        //TODO directly handle error in stream if there is more knowledge about how to do that
+        checkIfPathOrDockerGiven(bean);
+        // this check is important, otherwise an app could get overwritten
+        checkAppNotExists(bean.getName());
+
+        doCreate(bean, noStart);
+    }
+
+    private void doCreate(ApplicationBean bean, boolean noStart) throws CreationException {
+        try {
+            pushAppManifest(bean, noStart);
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new CreationException("FAILED: " + e.getMessage());
```

Comment:
Message fof exception begins wiht: FAILED: ***
Problem in: CreateApplicationCommand#run{}
You catch the exception as following: 
 catch (CreationException e) {
                System.out.println("FAILED:" + e.getMessage());
                e.printStackTrace();
            }
which results in e.g.: FAILED:FAILED: app exists already

Reply:
done